js/views/userPageVw.js

```diff
@@ -1026,12 +1051,14 @@ module.exports = Backbone.View.extend({
     "use strict";
     var self = this;
     //var formData = new FormData(this.$el.find('#userPageImageForm')[0]);
-    var serverUrl = self.options.userModel.get('serverUrl');
-    var imageURI = self.$el.find('#image-cropper').cropit('export', {
-      type: 'image/jpeg',
-      quality: 0.75,
-      originalSize: false
-    });
+    var serverUrl = self.options.userModel.get('serverUrl'),
+        imageURI = self.$el.find('#image-cropper').cropit('export', {
+          type: 'image/jpeg',
+          quality: 0.75,
+          originalSize: false
+        }),
+        request;
```

Comment:
It doesn't seem like the request variable is used in this function. What is it for?